src/components/elements/LikeButton.js

```diff
@@ -0,0 +1,28 @@
+import classes from "./LikeButton.module.css";
+import "bootstrap-icons/font/bootstrap-icons.css";
+import LikesContext from "../store/Likes";
+import { useContext } from "react";
+
+function LikeButton(props) {
+  const likeContext = useContext(LikesContext);
+
+  const postIsLiked = likeContext.isLiked(props.id);
+
+  function toggleLikeHandler() {\
+    if (postIsLiked) {
```

Comment:
Maybe this can be shorter if you used the shorthand if
postIsLiked ? likeContext.removeLike(props.id) : likeContext.addLike({id: props.id})
just a style suggestion, not mandatory

Reply:
Agree, have used it later in the doc so why not stay consistent